pages/common/zed.md

```diff
@@ -0,0 +1,24 @@
+# zed
```

Comment:
How prevalent is zed vs zeditor as binary names? Consider creating an alias.

Reply:
Alright. Then create an alias page
https://github.com/tldr-pages/tldr/blob/main/contributing-guides/translation-templates/alias-pages.md